tonic/functional/to_bina_rep.py

```diff
@@ -0,0 +1,68 @@
+import numpy as np
+import math
+
+
+def to_bina_rep_numpy(
+    events: np.ndarray,
+    to_frame_transform,
+    n_frames: int = 1,
+    n_bits: int = 8,
+):
+    """Representation that takes T*B binary event frames to produce a sequence of T frames of N-bit numbers.
+    To do so, N binary frames are interpreted as a single frame of N-bit representation. Taken from the paper
+    Barchid et al. 2022, Bina-Rep Event Frames: a Simple and Effective Representation for Event-based cameras
+    https://arxiv.org/pdf/2202.13662.pdf
+    Parameters:
```

Comment:
I'm not 100% sure but I think for the docs to render this properly it needs one line of whitespace above `Parameters:` and `Returns:`

Reply:
You are completely right. I'm going to fix that.